### PR TITLE
Upgrade belongs_to

### PR DESCRIPTION
**Pull request by yakloinsteak**

### CODE REVIEW

File: app/models/grda_warehouse/client_file.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ class ClientFile < GrdaWarehouse::File
 
     include ArelHelper
 
-    belongs_to :client, class_name: 'GrdaWarehouse::Hud::Client'
+    belongs_to :client, class_name: 'GrdaWarehouse::Hud::Client', optional: true
```
Comment (eanders) [inline app/models/grda_warehouse/client_file.rb]: This should not be optional, though I'm not sure it matters.

Comment (yakloinsteak) [reply app/models/grda_warehouse/client_file.rb]: I'm happy to remove any of these that you know to be non-optional. I'll check if it's not null in the database. I'm thinking any with not null can be optional: false, right?

Comment (eanders) [reply app/models/grda_warehouse/client_file.rb]: I'm about 2/3 of the way through removing the ones I'm 99% sure don't need to be there.

Comment (yakloinsteak) [reply app/models/grda_warehouse/client_file.rb]: Let me know when you're done, and I'll deploy to QA again

Comment (yakloinsteak) [reply app/models/grda_warehouse/client_file.rb]: ...and sync with release-8/change the branch we're merging into.

Comment (eanders) [reply app/models/grda_warehouse/client_file.rb]: done

**Comment by yakloinsteak** [comment]
@eanders not sure what's up with the specs after I merged in release-8 to make this mergeable.

**Comment by eanders** [comment]
@yakloinsteak, looks like one of those optionals was required.  My commit broke it too.  I'll see if I can track down which.

**Comment by yakloinsteak** [comment]
@eanders Want me to track down the cause?

**Comment by eanders** [comment]
I was just about to.  I'll take a stab at getting tests passing.

**Pull request merged**